### PR TITLE
Fix/myz 8719 remove idp login components

### PR DESCRIPTION
**Pull request by agiua**
# fix: remove deprecated components

### Priority

- [ ] 1 - Highest
- [ ] 2 - High
- [ ] 3 - Medium
- [x] 4 - Low
- [ ] 5 - Lowest
- [ ] 6 - Not a Priority

### Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Component (add a Component as approved by Design System)
- [ ] Docs (add documentation)
- [ ] Chore (changes that adds small enhancement)
- [ ] Breaking (fix or feature that would cause existing functionality to not work as expected)

## 

## Component and Fix approval flow to Master branch

A Pull Request to be merged must have two approvals, one from a member of the product team who developed it and another from a member of the dst dev team other than the team representative.

### CODE REVIEW

**Comment by francesco-santi-1** [comment]
Ci sono degli `<z-app-topbar>` rimasti in alcuni file di Storybook.

**Comment by francesco-santi-1** [comment]
E poi ci sono `<z-app-topbar>` e `<z-user-dropdown>` in due file di test: file-upload.html e notification.html.

**Review by francesco-santi-1 — APPROVED** [review]

File: src/components/navigation/z-app-header/index.stories.mdx [hunk 1]
```diff
@@ -1110,17 +1110,6 @@ import "./index";
   <Story name="With topbar">
```
Comment (francesco-santi-1) [inline src/components/navigation/z-app-header/index.stories.mdx:1110]: Questa storia, visto che è la variante con la topbar, forse ha senso toglierla del tutto. Così non ha senso lasciarla.

**Comment by vpasquino** [comment]
Valutare anche la rimozione di `ZMenuDropdown`?

**Review by vpasquino — APPROVED** [review]

**Pull request merged**